vector/build.gradle

```diff
@@ -317,11 +317,6 @@ android {
     }
 }
 
-configurations {
-    // videocache includes a sl4j logger which causes mockk to attempt to call the static android Log
-    testImplementation.exclude group: 'org.slf4j', module: 'slf4j-android'
-}
```

Comment:
As I removed `videocache`, I thought I could remove this as well. But maybe double check with @ouchadam first. And I just see that it's still needed: https://github.com/vector-im/element-android/pull/4670/files#diff-f6dc61a00090b6fc204667d7934e656dbb78c8bb0dc14f95468dd57b4300a8caR177

Reply:
if `videocache` is removed then removing this is also fine :+1:  (the unit tests will fail if not)